recipes/log.c/all/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+cmake_minimum_required(VERSION 3.16)
+
+project(log.c
```

Comment:
Let's keep the cmake script as dense as possible and throw out the things not needed per see.
I would only do `peoject(log.c C)` here.

Reply:
Remove info, but keep VERSION and LANGUAGES